targets/ESP32/_nanoCLR/nanoFramework.Hardware.Esp32.Rmt/rmt_multi_stage_encoder.cpp

```
@@ -0,0 +1,210 @@
+    mstep->loopCount = loopCount;
+    mstep->dataRequired = true;
+    mstep->data = NULL;
+    mstep->dataLen = 0;
+
+    multi_encoder->step++;
+
+err:
+    return ret;
+}
+
+esp_err_t rmt_add_copy_encoder(
```

Cppcheck (2.21.0)

[style] The function 'rmt_add_copy_encoder' is never used.

(unusedFunction)